@navikt/navno/css/card-micro.css

```
@@ -0,0 +1,56 @@
+:root {
+  --navds-card-micro-color-background-focus: var(--navds-global-color-white);
+  --navds-card-micro-color-background-hover: var(--navds-global-color-white);
+  --navds-card-micro-color-border-hover: var(--navds-global-color-blue-500);
+  --navds-card-micro-color-border-active: var(--navds-global-color-blue-500);
+  --navds-card-micro-color-border: transparent;
+  --navds-card-micro-color-shadow-focus: var(--navds-global-color-blue-800);
+  --navds-card-micro-color-text-active: var(--navds-global-color-white);
+  --navds-card-micro-color-text-focus: var(--navds-global-color-blue-500);
+  --navds-card-micro-color-text-hover: var(--navds-global-color-blue-500);
+  --navds-card-micro-color-text: var(--navds-global-color-gray-900);
+}
+
+.navds-card-micro {
+  display: inline-block;
+  padding: var(--navds-spacing-3) 0;
+}
+
+/* Ensure adjecent MicroCards are spaced between */
+.navds-card-micro:not(:last-of-type) {
+  margin: 0 var(--navds-spacing-2) 0 0;
+}
+
+.navds-card-micro:focus {
+  outline: 0;
+}
+
+.navds-card-micro .navds-card__bed {
+  border-radius: var(--navds-spacing-4);
+  color: var(--navds-card-micro-color-text);
+  display: inline-block;
+  font-size: var(--navds-font-size-small);
+  padding: 0 var(--navds-spacing-2);
+  text-decoration: none;
+  transition: all 0.3s;
+  width: auto;
+}
+
+.navds-card-micro:hover:not(:focus):not(:active) .navds-card__bed {
+  background-color: var(--navds-card-micro-color-background-hover);
+  box-shadow: 0 0 0 1px var(--navds-product-card-color-border-hover);
+  color: var(--navds-card-micro-color-text-hover);
+}
+
+.navds-card.navds-card-micro:focus:not(:active) .navds-card__bed {
+  background-color: var(--navds-card-micro-color-background-focus);
+  box-shadow: 0 0 0 3px var(--navds-card-micro-color-shadow-focus);
```

Vi har en token `--navds-shadow-focus` som er `0 0 0 3px ...blue-800`

```suggestion
  box-shadow: var(--navds-shadow-focus);
```